app/src/sharedTest/java/org/oppia/app/profile/AddProfileActivityTest.kt

```diff
@@ -81,6 +82,15 @@ class AddProfileActivityTest {
     }
   }
 
+  @Test
+  fun testAddProfileActivity_createPin_checkOpensProfileActivity() {
+    ActivityScenario.launch(AddProfileActivity::class.java).use {
+      onView(allOf(withId(R.id.checkbox_pin))).perform(
+        click()
+      )
+      onView(withId(R.id.input_pin)).check(matches(isDisplayed()))   }
+  }
+
```

Comment:
Add one more test case, which makes sure that by default input_pin is not visible.